public/app/features/dashboard-scene/sharing/ExportButton/ExportAsJson.tsx

```diff
@@ -28,6 +31,13 @@ function ExportAsJsonRenderer({ model }: SceneComponentProps<ExportAsJson>) {
     return JSON.stringify(json, null, 2);
   }, [isSharingExternally]);
 
+  const onClickDownload = async () => {
+    await model.onSaveAsFile();
+    const message = t('export.json.download-successful_toast_message', 'Your JSON has been downloaded');
+    dispatch(notifyApp(createSuccessNotification(message)));
+    dashboard.closeModal();
```

Comment:
why should we close the drawer? None of the others is closed when an action is fired and it's not the current behavior, neither

Reply:
@natydej has it for export pdf and wanted to extended it to json. But maybe we need to further discuss it

Reply:
@lucychen-grafana after discussion with @juanicabanas we consider that closing the drawer in this case is not necessary as in the “save” dashboard drawer, as people may want to select another option after clicking "download", such as copying the JSON file. We should close the drawer only if there is no other action to perform. Therefore, we considered leaving the drawer open for the PDF option as well.

Reply:
@natydej sounds good! But we want to keep the toast message to let them know that download has been successful right?